partials/footer.hbs

```diff
@@ -0,0 +1,53 @@
+<footer class="page-footer font-small blue pt-4">
+  <div class="container-fluid text-center text-md-left">
+    <div class="row">
+      <div class="col-md-4 mt-md-0 mt-3">
+        <h2 class="">Contact us</h2>
+        <ul class="list-unstyled">
+          <li>
+            <a href="mailto:keepsafe@consento.org">Investors</a>
```

Comment:
@dkastl do you think these should be different mail addresses?
@Torgian Maybe add a ?subject=?